pkg/okctl/okctl.go

```diff
@@ -576,11 +543,6 @@ func (o *Okctl) CoreServices() (core.Services, error) {
 		run.NewKubeRun(o.CloudProvider, o.CredentialsProvider.Aws()),
 	)
 
-	awsIamAuth, err := o.BinariesProvider.AwsIamAuthenticator(awsiamauthenticator.Version)
-	if err != nil {
-		return core.Services{}, err
-	}
```

Comment:
Why remove this?

Reply:
It's moved up to line 487?

Reply:
My bad. Shouldn't read big PRs before 0900